src/main/kotlin/Network.kt

```diff
@@ -0,0 +1,29 @@
+class Network() {
+val capacity = 5
+    fun tallyUpCars(ListOfInterestToDrive:MutableList<Int>) : Int {
+        var ListOfCarsThatWantToDrive:MutableList<Int> = mutableListOf()
+        for (interestToDrive in ListOfInterestToDrive){
+            if (interestToDrive==1){
+                ListOfCarsThatWantToDrive.add(0)
+            }
+
+        }
+        val numberOfCarsThatWantToDrive = ListOfCarsThatWantToDrive.size
+        return numberOfCarsThatWantToDrive
+    }
+
```

Comment:
Absolutely, but the task says explicitly, that the file should contain the interest to drive AND the delay.. That's what I can't figure out clearly.